run/config.web.toml

```diff
@@ -5,6 +5,10 @@ ip = ""
 port = 0
 server_url = ""
 registry_url = ""
+stop_file = "stopwebui"
+user = ""
+email = ""
+#log_level = "debug"
```

Comment:
a) see my earlier comment about maybe not being optional
b) lowercase "debug" is not one of the allowed values in the docs
c) `#foo` -> `# foo` please